sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs

```diff
@@ -8,7 +8,7 @@
 namespace Azure.Data.AppConfiguration
 {
     /// <summary>
-    /// Options that allow to configure the management of the request sent to the service
+    /// Options that allow to configure the management of the request sent to the service.
```

Comment:
Let's change this to "Options that allow users to configure the requests sent to the App Configuration service."